python/lsst/jointcal/testUtils.py

```diff
@@ -229,7 +229,7 @@ def fillCatalog(schema, num, bbox,
     table = lsst.afw.table.SourceTable.make(schema)
     table.defineCentroid('centroid')
     table.defineShape('shape')
-    table.defineInstFlux(instFluxKeyName)
+    table.defineCalibFlux(instFluxKeyName)
```

Comment:
Based on code below, such as `star.setInstFluxAndErr(record.getCalibInstFlux(), record.getCalibInstFluxErr())`, it appears that calibrated flux is calibrated *instrument* flux, so shouldn't this be named `defineCalibInstFlux`?

Reply:
The table method really should be `defineCalibFlux`, but you bring up a good point: I've changed `instFluxKeyName` to `fluxKeyName`, since it's the name of the field (which gets e.g. `"_flux"`appended).

Reply:
Thanks!